src/test/kotlin/JiraPerformanceComparisonProcessOnlyIT.kt

```diff
@@ -0,0 +1,79 @@
+import com.atlassian.performance.tools.jiraactions.api.ActionMetric
+import com.atlassian.performance.tools.jiraactions.api.ActionResult
+import com.atlassian.performance.tools.jiraactions.api.format.MetricCompactJsonFormat
+import com.atlassian.performance.tools.jiraactions.api.format.MetricJsonFormat
+import com.atlassian.performance.tools.report.api.FullReport
+import com.atlassian.performance.tools.report.api.FullTimeline
+import com.atlassian.performance.tools.report.api.Timeline
+import com.atlassian.performance.tools.report.api.result.EdibleResult
+import com.atlassian.performance.tools.report.api.result.RawCohortResult
+import com.atlassian.performance.tools.workspace.api.RootWorkspace
+import org.junit.Test
+import java.io.File
+import java.nio.file.Paths
+
+class JiraPerformanceComparisonProcessOnlyIT {
```

Comment:
It's a bit wordy. We can avoid these hard to remember names by simplifying. Most of the name tries to explain what it _does not do_. Let's focus on what it actually does, e.g. `ExistingResultsIT.shouldReport()`